### PR TITLE
add okta custom auth server

### PR DESCRIPTION
**Pull request by kjiang14**
Adding support for Okta custom authorization server as described in this PR: https://github.com/SocialiteProviders/Okta/pull/2

### CODE REVIEW

**Review by atymic — COMMENTED** [review]
Could you link to the docs covering this please?

**Comment by atymic** [comment]
cc @kjiang14

**Comment by robiiinos** [comment]
@atymic mmh, wait... this PR add custom auth server to Okta, but the updated README is for the YouTube Provider...

**Comment by atymic** [comment]
🤦 

I'll fix that shortly haha. Good catch.

File: src/Okta/Provider.php [hunk 1]
```diff
@@ -44,36 +44,44 @@ protected function getOktaUrl()
         return $this->getConfig('base_url');
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function getAuthServerId()
+    {
+        return $this->getConfig('auth_server_id', 'default');
```
Comment (gavinr) [inline src/Okta/Provider.php:52]: This should not default to `default` - per [this doc](https://support.okta.com/help/s/article/400-error-on-authentication-to-Okta-using-OIDC?language=en_US), many okta users should not have `default` for the `auth_server_id`. And you can't override this to an empty string in the config - if you send in an empty string in the config, `default` is set.

Comment (gavinr) [reply src/Okta/Provider.php:52]: Nevermind, looks like this was fixed in 8bdcb1ab129638daa11882168b80d3e4609d559b

**Pull request merged**